src/Backend/Modules/Blog/DataFixtures/LoadBlogCategories.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Backend\Modules\Blog\DataFixtures;
+
+class LoadBlogCategories
+{
+    public function load(\SpoonDatabase $database)
```

Comment:
slightly based on Doctrine's datafixtures, to make it easier to replace these later.